index.js

```diff
@@ -12,7 +12,8 @@ class Task {
 class TaskList {
   constructor() {
     this.tasks = this.loadTasksFromLocalStorage() || [];
-    this.displayTasks();
+    this.filteredTasks = [...this.tasks];
+    this.filterTasks();
```

Comment:
I don't like this idea as we store the tasks twice in some cases(Image we have a lot of tasks with its data/images, etc..).
The request is very simple, store the filter only and not the filtered tasks 😁
And starts rendering the tasks based on the filter value,